src/intl/en/learn-quizzes.json

```diff
@@ -324,5 +324,52 @@
   "h005-c-label": "Eth1",
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
-  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain."
+  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "i001-prompt": "What is required to 'run a node'?",
+  "i001-a-label": "Run software, with modest hardware, while online",
+  "i001-a-explanation": "Operating a node consists of running software that communicates using the language of the Ethereum protocol with other computers doing the same. This software downloads a copy of the Ethereum blockchain, verifies the validity of every block, then keeps it up-to-date with new blocks and transactions, while helping others download and update their own copies.",
+  "i001-b-label": "Deposit 32 ETH to earn rewards",
+  "i001-b-explanation": "This is a requirement for staking—the process of becoming an active participant in network consensus. This is not required to simply run a sovereign copy of the blockchain, which requires NO ETH.",
+  "i001-c-label": "Operate powerful ASIC mining machines to reach network consensus",
+  "i001-c-explanation": "Although Ethereum previously used mining with powerful computers to reach consensus, this process has been replaced entirely by staking. Neither mining in the past, nor staking currently, are required to simply operate a sovereign copy of the blockchain.",
+  "i001-d-label": "Work full time in blockchain infrastructure",
+  "i001-d-explanation": "Software tooling has continued to improve over time making running a node from home as a novice much more approachable. Working full-time in blockchain infrastructure is by no means a requirement to get involved.",
+  "i002-prompt": "How much ETH must be staked to operate a node?",
+  "i002-a-label": "0",
+  "i002-a-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to operating a staking validator as part of a node setup, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
+  "i002-b-label": "8",
+  "i002-c-label": "16",
+  "i002-d-label": "32",
+  "i002-d-explanation": "Operating an Ethereum node does NOT require any ETH. In contrast to the 32 ETH required to activate a staking validator that participates directly in network consensus, anyone is free to run client software and sync their own sovereign copy of the blockchain—no ETH required.",
+  "i003-prompt": "How does running an Ethereum node benefit you as a user?",
```

Comment:
```suggestion
  "i003-prompt": "What benefits do you get from running your own node?",
```

Reply:
Yeah, was trying to contrast benefits to users vs the whole network, but didn't end up writing that question... I like this change